components/frontend/src/subject/Subject.jsx

```diff
@@ -1,5 +1,3 @@
-import "./Subject.css"
```

Comment:
This file seems to have already been deleted before, was this not throwing an error?

Reply:
The file was renamed.